### PR TITLE
Don't check for Datacenter class

### PR DESCRIPTION
**Pull request by agrare**
The hash isn't an ActiveRecord object yet so we can't use
f.kind_of?(Datacenter)

### CODE REVIEW

File: app/models/manageiq/providers/redhat/infra_manager/event_parser.rb [hunk 1]
```diff
@@ -78,8 +78,7 @@ def self.ems_ref_from_object_in_event(data)
   end
 
   def self.parse_new_target(full_data, message, ems, event_type)
-    folders = parse_new_folders(full_data.data_center)
-    dc      = folders.detect { |f| f.kind_of?(Datacenter) }
+    dc, vm_folder, host_folder = parse_new_folders(full_data.data_center)
```
Comment (Fryguy) [inline app/models/manageiq/providers/redhat/infra_manager/event_parser.rb]: optionally you can also do...
```ruby
dc, *folders = parse_new_folders(full_data.data_center)
...
:folders => [dc, *folders]
```

Comment (agrare) [reply app/models/manageiq/providers/redhat/infra_manager/event_parser.rb]: :+1: done

**Comment by agrare** [comment]
@borod108 @mwperina master has been red since Jan 21st

**Comment by chessbyte** [comment]
Merging with test failures as they are not related to this PR.

**Pull request merged**